daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -2773,6 +2772,41 @@ object EngineTest {
   private def suffix(tx: Tx.Transaction) =
     data.assertRight(tx.suffixCid(_ => dummySuffix))
 
+  private[this] case class ReinterpretState(
```

Comment:
I somewhat wonder if we can and should use the scenario ledger here. This seems like we’re partially reimplementing what that does.
Let’s leave it out of this PR though.

Reply:
Good point.